Description: Change example to make it compatible with the Debian init scripts
  per default.
Author: Christian Hammers <ch@debian.org>

--- old/vtysh/vtysh.conf.sample.orig	2004-10-30 23:07:40.000000000 +0200
+++ new/vtysh/vtysh.conf.sample	2004-10-30 23:08:24.000000000 +0200
@@ -1,7 +1,7 @@
 !
 ! Sample configuration file for vtysh.
 !
-!service integrated-vtysh-config
+service integrated-vtysh-config
 !hostname quagga-router
-!username root nopassword
+username root nopassword
 !
